SRC/classes/StudentQuery.cls

```diff
@@ -0,0 +1,10 @@
+public class StudentQuery {
+    public StudentQuery(){
+        List<Student__c> StudentList = [select id, Name,Roll_Number__c,Father_Name__c,Father_Mobile_Number__c,
```

Comment:
Always Use Query Indentation 
Ex: Select or SELECT it shouldn't be select, And all field names should be start with capital Letters Id, Name and Variables camelCase ex: studentList